tests/packages/test_file_dependency.py

```diff
@@ -22,7 +22,7 @@
 
 def test_file_dependency_wrong_path() -> None:
     with pytest.raises(ValueError):
-        FileDependency("demo", DIST_PATH / "demo-0.2.0.tar.gz")
+        FileDependency("demo", DIST_PATH / "demo-0.2.0.tar.gz").full_path
```

Comment:
I think these tests should be testing each individual failure mode, but this was good enough for years so I suppose it will do